agent/tool-scripts/postprocess/unittests

```diff
@@ -203,9 +192,18 @@ function testfio {
 
     mkdir ${_testroot}/$1
     cp -r samples/$1/* ${_testroot}/$1/
+    result_dir=${_testroot}/$1/result/reference-result
 
-    pbench_lib_dir=$(pwd)/../../lib pbench_bspp_dir=$(pwd)/../../bench-scripts/postprocess \
-        ../../bench-scripts/postprocess/fio-postprocess ${_testroot}/$1/result/reference-result fio- default > ${_testroot}/$1/stdout 2> ${_testroot}/$1/stderr
+    pbench_lib_dir=$(pwd)/../../lib \
+        pbench_bspp_dir=$(pwd)/../../bench-scripts/postprocess \
+        ../../bench-scripts/postprocess/fio-postprocess ${result_dir} fio- default \
+        > ${_testroot}/$1/stdout 2> ${_testroot}/$1/stderr
+
+    result_lines=$(wc -l ${result_dir}/result.json | awk '{ print $1 }')
+    if (( result_lines > MAX_RESULT_LINES )); then
+        head -n $MAX_RESULT_LINES ${result_dir}/result.json >${result_dir}/result.json.trunc
+        mv ${result_dir}/result.json.trunc ${result_dir}/result.json
+    fi
```

Comment:
We should just do the `head` unconditionally -- it won't mind if the file is shorter than the max -- the `head` and the `mv` are probably no more expensive than the `wc` and the `awk`, and it saves traversing the file the second time in the max-exceeded case.